Assignments/Program.cs

```diff
@@ -1 +1,53 @@
-﻿Console.WriteLine("Hello, World!");
+﻿// ---------------------------------------------------------------------------------------
+// T12 - Pascal's Triangle
+// ---------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Spark;
+
+public class T12 {
+   public static void Main () {
+      Console.WriteLine ("\x1B[4m" + "Pascal Triangle Printer:-" + "\x1B[0m");
+      Console.Write ("\nEnter the length of the tringle (l <= 13): ");
+      int len = Convert.ToInt32 (Console.ReadLine ());
+      PascalTriangle (len < 14 ? len : 13);
+   }
+
+   /// <summary>Prints the Pascal's Traingle of given length</summary>
+   public static void PascalTriangle (int length) {
+      bool putSlash = length < 9;

```

Comment:
![image](https://github.com/vikrampg1710-tmm/Training/assets/86363064/4ec010ac-410e-4372-9289-29bdc752cede)
IMO, the triangle looks nicer without the slash. Benefit: no need for the calculation in line 39.
It would have been enough to print just the numbers, with appropriate offset